packages/nx-plugin/src/executors/build/schema.json

```diff
@@ -30,6 +30,21 @@
       "description": "Paths to global stylesheets to be included.",
       "default": []
     },
+    "stylePreprocessorOptions": {
+      "description": "Options to pass to style preprocessors.",
+      "type": "object",
+      "properties": {
+        "includePaths": {
+          "description": "Paths to include. Paths will be resolved to project root.",
```

Comment:
This mentions paths being resolved to project root, but the implementation in `createConfig` is joining the workspaceRoot, not the project root.

Reply:
Thanks for the comment.

I am not sure why the schema of **stylePreprocessorOptions** defined in **angular/executors/webpack-browser.json** is described relative to project root, but most of other paths are relative to workspace root.

Here is the one I should copy from **angular/executors/browser-esbuild.json**
```
"stylePreprocessorOptions": {
  "description": "Options to pass to style preprocessors.",
  "type": "object",
  "properties": {
    "includePaths": {
      "description": "Paths to include. Paths will be resolved to workspace root.",
      "...
    }
  },
  ...
},
```

From my thought, it looks not strange to set the path outside the project. If we leverage and keep the consistence of the path design among other executors, it will benefit people to migrate from webspack or esbuild more easily.

Weather to change the path design from relative to project root to workspace root or not, I will fix my mistake of inconsistence first.